assets/src/js/admin.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-import './admin.css';
+import '././sass/admin.scss';
```

Comment:
Can you change this to `import './sass/admin.scss';`?

Reply:
Ah! That was my mistake. But just to be clear:
Do you want the new Sass files under `src` or under `js`? I was thinking that it should be `koko-analytics > assets > src > sass`

Reply:
`assets > src > sass` works for me!